how-to/how-to-design-database-schema.md

```diff
@@ -0,0 +1,304 @@
+---
+title: How to design your Telerik Platform mobile app backend database
+description: An overview of NoSQL database vs relational database and how to structure your data in Telerik Platform.
+type: how-to
+page_title: Telerik Platform Schema Design
+slug: design-database-schema
+position:
+tags: backendservices, relations
+ticketid:
+pitsid:s
+
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>
+		{{site.bs-d}} ({{site.bs}}), <!--Data (Backend Services)-->
+	</td>
+  </tr>
+</table>
+
+## Description

```

Comment:
I feel like rewording these questions into an explanation about the purpose and content. For example: When designing the backend types and modeling the schema of your app's backend yo ufrequently need to take into acount the specifics and tools of non-relational database and how to optimize this....